string task
string image_name
sensor_msgs::Image image_data
